src/poliastro/czml/extract_czml.py

```diff
@@ -403,29 +187,43 @@ def add_ground_station(
                 "Invalid coordinates. Coordinates must be of the form [u, v] where u, v are astropy units"
             )
 
-        self._init_ground_station_packet_(self.gs_n, pos)
-        self._change_ground_station_params_(
-            id_name=id_name,
-            id_description=id_description,
-            label_fill_color=label_fill_color,
-            label_font=label_font,
-            label_outline_color=label_outline_color,
-            label_text=label_text,
-            label_show=label_show,
+        pckt = Packet(
+            id="GS" + str(self.gs_n),
+            description=id_description,
+            availability=self.start_epoch.value + "/" + self.end_epoch.value,
+            position=Position(cartesian=pos),
+            label=Label(
+                show=label_show,
+                text=label_text,
+                font=label_font if label_font is not None else "11pt Lucida Console",
+                fillColor=Color(rgba=label_fill_color)
+                if label_fill_color is not None
+                else None,
+                outlineColor=Color(rgba=label_outline_color)
+                if label_outline_color is not None
+                else None,
+            ),
+            billboard=Billboard(
+                image=PIC_GROUNDSTATION,
+                horizontalOrigin="CENTER",
+                verticalOrigin="CENTER",
```

Comment:
Same for `czml3.enums.VerticalOrigins.CENTER`

Reply:
However, again these are the defauls and probably don't need to be specified https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Billboard#properties